python/modules/IcePy/Communicator.cpp

```diff
@@ -328,17 +324,45 @@ communicatorDestroyAsync(CommunicatorObject* self, PyObject* /*args*/)
                 Py_XDECREF(self->wrapper);
                 self->wrapper = nullptr;
 
-                PyObject* err = PyErr_Occurred();
-                if (err)
+                PyObject* emptyArgs = PyTuple_New(0);
```

Comment:
Maybe at a reminder to upgrade to the empty tuple constant with 3.13?

Reply:
Not sure we have empty tuples all over the place, at some point when we increase the minimum supported version we would need to go over the new features.